deployer/hub.py

```diff
@@ -341,9 +345,10 @@ class Hub:
     A single, deployable JupyterHub
     """
 
-    def __init__(self, cluster, spec):
+    def __init__(self, cluster, spec, config_path):
```

Comment:
If this is the constructor for `Hub` class and we have a `Cluster` class - what does `config_path` refer to?

Here is a snippet from the Cluster class' constructor.

```python
    def __init__(self, spec, config_path):
        self.spec = spec
        self.config_path = config_path
        self.hubs = [Hub(self, hub_yaml, config_path) for hub_yaml in self.spec["hubs"]]
```

What do you think about being more verbose, and let `config_path` be named `cluster_config_path`, or perhaps rely on the cluster object like `self.cluster.config_path` etc?

Reply:
Yeah, this telling to how inexperienced I am at working with Python classes 😁 Relying on the Cluster object to propagate the path the the cluster.yaml file didn't occur to me as a possibility

Reply:
Done in [bacd6e0](https://github.com/2i2c-org/infrastructure/pull/1027/commits/bacd6e04b17131a1501c38bed0e687e8ef3fe01c)